src/Executables/Atlas/1. Software/Install Software.ps1

```diff
@@ -82,6 +82,9 @@ $Form.Controls.Add((generate_checkbox "Discord Canary" "discord-canary"))
 # https://community.chocolatey.org/packages/steam
 $Form.Controls.Add((generate_checkbox "Steam" "steam"))
 
+# https://community.chocolatey.org/packages/legendary
+$Form.Controls.Add((generate_checkbox "legendary" "legendary"))
```

Comment:
I don't think that many people will use raw Legendary from the CLI, much more people would use Heroic instead, but that doesn't seem to have a Chocolatey package. https://heroicgameslauncher.com/

Reply:
Yes, i considered that, but as you mentioned is not available. That's why there is a solution: 

https://community.chocolatey.org/packages/rare

Reply:
@he3als can you approve now, get this pr closed